Library/Account.cs

```diff
@@ -463,6 +463,7 @@ public override int GetHashCode()
     public sealed class Accounts
     {
         internal const string UrlPrefix = "/accounts/";
+        private static readonly QueryStringBuilder Build = new QueryStringBuilder();
```

Comment:
Can ignore this. Might get rid of this builder in favor of using `NamedValueCollection`